components/nimbus/src/enrollment.rs

```diff
@@ -648,7 +649,7 @@ impl<'a> EnrollmentsEvolver<'a> {
             &prev_rollouts,
             &next_rollouts,
             &ro_enrollments,
-            event_store,
+            event_store.clone(),
```

Comment:
The function that called this also `clone()` the event store so now it's been cloned twice before being used at all